swap_meet/vendor.py

```diff
@@ -0,0 +1,78 @@
+from swap_meet.item import Item
+
+class Vendor:
+    def __init__(self, inventory=None):
+        if inventory == None:
+            self.inventory = []
+        # too strict a condition? replace with 'else:\n self.inventory = inventory'?
+        elif type(inventory) == list: 
+            self.inventory = inventory
+    
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+    
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        else:
+            return False
+
+    def get_by_category(self, category):
+        hold_relevant_items = []
+        list_to_check = self.inventory
+        
+        for item in list_to_check:
+            if item.category == category:
+                hold_relevant_items.append(item)
+        
+        return hold_relevant_items
+
+    def swap_items(self, Vendor, my_item, their_item):
+        vendor_prime = self
+        vendor_second = Vendor
+
+        if my_item in vendor_prime.inventory and their_item in vendor_second.inventory:
+            vendor_prime.add(their_item)
+            vendor_prime.remove(my_item)
+            vendor_second.add(my_item)
+            vendor_second.remove(their_item)
+            return True
+        else: 
+            return False
+
+    def swap_first_item(self, Vendor):
```

Comment:
I suggest not using `Vendor` as a variable name.  Instead use `other_vendor`.  `Vendor` is the name of a class.